rust_src/build.rs

```diff
@@ -341,7 +341,7 @@ fn handle_file(mod_path: &PathBuf) -> Result<Option<ModuleData>, BuildError> {
                     e.kind(),
                     format!("Failed to open {}: {}", mod_info.path.to_string_lossy(), e),
                 )
-                .into())
+                .into());
```

Comment:
? Where did this come from? Is your rustfmt not matching ours?

Reply:
it did come from rustfmt. I just installed `rustfmt-nightly` seems I seem to need that with the toolchain configured in the repo. Unfortunately I did make sure to only add the file I touched, but then I reflexibly did `git commit -am` so I still sent everything over. I'll force-push with the correct commit.

Reply:
You want `rustup component add rustfmt-preview`.